pom.xml

```diff
@@ -21,7 +21,7 @@ Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.apache.synapse.inbound</groupId>
     <artifactId>org.wso2.carbon.inbound.iso8583</artifactId>
-    <version>1.0.2-SNAPSHOT</version>
+    <version>1.0.1</version>
```

Comment:
how do you go from 1.0.2 snapshot to 1.0.1?

Reply:
We need to release the inbound.